packages/core/src/2d/sprite/Sprite.ts

```diff
@@ -45,22 +49,10 @@ export class Sprite extends RefObject {
   set texture(value: Texture2D) {
     if (this._texture !== value) {
       this._texture = value;
-      this._setDirtyFlagTrue(DirtyFlag.positions);
+      this._dispatchSpriteChange(SpritePropertyDirtyFlag.texture);
     }
   }
 
-  /**
-   *  Bounding volume of the sprite.
-   *  @remarks The returned bounds should be considered deep-read-only.
-   */
-  get bounds(): Readonly<BoundingBox> {
```

Comment:
Bounds can't delete, this is a unified concept setting,like Mesh.bounds,Mesh.MeshRenderer.bounds